logger/logfactory_test.go

```diff
@@ -23,6 +23,7 @@ func TestConcurrentGetLoggerOk(t *testing.T) {
 			defer func() {
 				if r := recover(); r != nil {
 					recovered = r
+					fmt.Println(recovered)
```

Comment:
We should remove print statement.

Reply:
This is test case, it should be ok :) but I removed it